core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorTableSchema.java

```diff
@@ -41,12 +44,18 @@ public List<ColumnSchema> getColumns()
         return columns;
     }
 
+    public List<String> getCheckConstraints()
```

Comment:
I'm not sure what class should be used for representing CHECK constraints in SPI. Needs suggestion.